hawkular-component/src/main/java/org/hawkular/metrics/security/InfluxAuthHttpHandler.java

```diff
@@ -68,9 +73,7 @@ public void handleRequest(HttpServerExchange httpServerExchange) throws Exceptio
         HeaderMap requestHeaders = httpServerExchange.getRequestHeaders();
 
         String databaseName = matcher.group(1);
-        try {
```

Comment:
The regex above is not meant to replace this try/catch. It might be the case that it matches the regex but still is not spec-compliant. In such case, Accounts will throw a RuntimeException, so, it's a matter of taste now :)